pkg/backporter/backporter_test.go

```diff
@@ -199,15 +182,137 @@ func TestGetClonesHandler(t *testing.T) {
 			}
 			req.URL.RawQuery = q.Encode()
 			rr := httptest.NewRecorder()
-			handler := unwrapper(GetClonesHandler(fake))
+			handler := unwrapper(ClonesHandler(fake, allTargetVersions))
 			handler.ServeHTTP(rr, req)
 			if status := rr.Code; status != tc.results.statusCode {
 				t.Errorf("testcase '%v' failed: getbug returned wrong status code - got %v, want %v", tc, status, tc.results.statusCode)
 			}
-			if resp := rr.Body.String(); resp != tc.results.htmlPage {
-				t.Errorf("Response differs from expected by: %s", diff.StringDiff(resp, tc.results.htmlPage))
+			var buf bytes.Buffer
+			if err := tc.results.tmplt.Execute(&buf, tc.results.data); err != nil {
+				t.Errorf("Unable to render template: %v", err)
+			}
+			subPage := buf.String()
+			var expResponse string
+			if tc.results.statusCode == http.StatusOK {
+				expResponse = fmt.Sprintf(htmlPageStart, "Clones") + subPage + htmlPageEnd
+			} else {
+				expResponse = fmt.Sprintf(htmlPageStart, "Not Found") + subPage + htmlPageEnd
+			}
+			if resp := rr.Body.String(); resp != expResponse {
+				t.Errorf("Response differs from expected by: %s", diff.StringDiff(resp, expResponse))
 			}
 		})
 	}
 
 }
+
+func TestCreateCloneHandler(t *testing.T) {
+	fake := &bugzilla.Fake{}
+	fake.Bugs = map[int]bugzilla.Bug{}
+	fake.BugComments = map[int][]bugzilla.Comment{}
+
+	bug1Create := &bugzilla.BugCreate{
```

Comment:
Can we give this a somewhat useful name instead of enumerating, like for example `toBeCloned`?